src/m365/aad/commands/o365group/o365group-user-list.ts

```diff
@@ -90,25 +90,23 @@ class AadO365GroupUserListCommand extends GraphCommand {
     }
   }
 
-  private getOwners(logger: Logger, groupId: string): Promise<User[]> {
+  private async getOwners(groupId: string): Promise<User[]> {
```

Comment:
You removed the logger here, but it seems useful to me if we put a verbose logging line in instead.